source/fe/fe_tools.cc

```diff
@@ -2519,7 +2519,12 @@ namespace FETools
   }
 
 
-
+  template <int dim>
+  FiniteElement<dim> *
+  get_fe_from_name (const std::string &parameter_name)
+  {
+    return get_fe_by_name<dim,dim> (parameter_name);
+  }
 
 
```

Comment:
You're right... In the latest commit we have three lines between all functions that do not just differ by template specialization.